src/main/resources/application.properties

```diff
@@ -19,6 +19,7 @@ spring.liquibase.change-log=classpath:liquibase/db.changelog-master.xml
 ### Hibernate Properties
 # The SQL dialect makes Hibernate generate better SQL for the chosen database
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
+spring.jpa.properties.hibernate.globally_quoted_identifiers=true
```

Comment:
In which cases you get a problem that this property covers?

Reply:
In case when SQL query contains a keyword "show". It can be solved by adding @Column(name = "\"show\"") to field of Phone class, but then tests are failed. Maybe because of h2 in tests, but for testing this field should not be quote, so I didn't add this property to test/resources/application.property. 
Also I think not using keywords can help.